stripe/api_requestor.py

```diff
@@ -357,7 +358,8 @@ def interpret_response(self, rbody, rcode, rheaders):
                 rbody, rcode, rheaders)
         if not (200 <= rcode < 300):
             self.handle_error_response(rbody, rcode, resp, rheaders)
-        return resp
+
+        return StripeResponse(rbody, rcode, rheaders)
```

Comment:
One minor optimization is that you could instantiate the `StripeResponse` in the `try` clause, to avoid parsing the JSON twice.

Reply:
✅ done!